src/theme.tsx

```diff
@@ -1,20 +1,89 @@
 import { createTheme, ThemeProvider } from "@mui/material";
 import React from "react";
 
-export const theme = createTheme({
+import { green, purple, gray, white, red, orange, blue, black } from "./colors";
+
+export const grayscaleTheme = createTheme({
   typography: {
     fontFamily: '"Inter", sans-serif'
   },
   palette: {
     primary: {
-      main: "#C4C4C4"
+      main: "#C4C4C4",
+      contrastText: white
     },
     secondary: {
-      main: "#7E7E7E"
+      main: "#7E7E7E",
+      contrastText: white
     }
   }
 });
 
+const baseTheme = createTheme({
+  typography: {
+    fontFamily: '"Inter", sans-serif'
+  },
+  palette: {
+    primary: {
+      light: green[300],
+      main: green[400],
+      dark: green[700],
+      contrastText: white
+    },
+    secondary: {
+      light: gray[300],
+      main: gray[400],
+      dark: gray[700],
+      contrastText: white
+    },
+    warning: {
+      main: orange,
+      contrastText: white
+    },
+    error: {
+      main: red,
+      contrastText: white
+    },
+    info: {
+      main: blue,
+      contrastText: white
+    },
+    success: {
+      main: green[500],
+      contrastText: white
+    },
+    mode: "light"
```

Comment:
It is optional but I added it because in the design mockups there were some examples with a dark background so I'm not sure if that something we want to explore afterwards